logfire/_internal/integrations/aiohttp_server.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+import warnings
+
+warnings.filterwarnings(
```

Comment:
aiohttp has deprecated the underlying Application which now gives off this warning upon subclassing. 
The fix lies in OpenTelemetry AioHttpServerInstrumentor. 

Any thoughts on how to handle this better?